code/game/gamemodes/blob/powers.dm

```diff
@@ -7,22 +7,23 @@
 
 // Power verbs
 
-/mob/camera/blob/proc/place_blob_core(point_rate, placement_override)
+/mob/camera/blob/proc/place_blob_core(point_rate, placement_override , pop_override = FALSE)
 	if(placed && placement_override != -1)
 		return 1
 	if(!placement_override)
-		for(var/mob/living/M in range(7, src))
-			if("blob" in M.faction)
-				continue
-			if(M.client)
-				to_chat(src, "<span class='warning'>There is someone too close to place your blob core!</span>")
-				return 0
-		for(var/mob/living/M in view(13, src))
-			if("blob" in M.faction)
-				continue
-			if(M.client)
-				to_chat(src, "<span class='warning'>Someone could see your blob core from here!</span>")
-				return 0
+		if(!pop_override)
```

Comment:
why not just use placement override? not using it would make abusing the object checks to become an overmind without placing on the spot possible, which I doubt you want.

Reply:
Not sure what you mean ? This is just to separate density checks (which i still want to happen for human popping) and visibility checks (which i do not want since it's the funniest part of human popping) without forcing the random spot.